### ISSUE TITLE
Experiencing with my Sennheiser IE 300 + 32Bit@384khz DAC hissing Noises

### ISSUE THREAD

**Issue by christianmende** [Pittvandewitt/Wavelet#140]
And my next issue:

That app does a very nice job for equalizing my reference IEMs. Problem: The hissing comes always with these pair of headphones (sadly a part of it's sound signature) and thought if I use AutoEQ it would nearly vanish, but certainly it does not. Is there a way to compensate this a bit more with the eq's settings? The big bummer actually is that i like bright presets, so  high frequencies are boostet anyways, but it hisses even without a preset :(

Regards

Little note:
I remember, when i use the bit-perfect mode in neutron and told neutron to convert all audio in DSD256 (supported by my DAC) it worked way better and was way more comfortable for my ears. Just a little inspiration for future updates ;))

**Comment by christianmende** [comment]
Update:

My Sennheiser IE 300 decided to fail nearly simultaneously on both sides. But besides that: It was actualy my fault, using cheap Aliexpress DACs... Now I am owning the IE 600, which I drive using a FiiO BTR5 2021 and this app is just heaven for equalization on Android.

I ordered in the past used replacements just to get sure my old DAC didn't fry my phones, but as I was tired of this DAC and I got the new one, I was stunned, because of the lower THD.
